manim/mobject/types/vectorized_mobject.py

```diff
@@ -1023,6 +1049,50 @@ def point_from_proportion(self, alpha: float) -> np.ndarray:
 
             current_length += length
 
+    def proportion_from_point(
+        self,
+        point: typing.Iterable[typing.Union[float, int]],
+    ) -> float:
+        """Returns the proportion along the path of the :class:`VMobject`
+        a particular given point is at.
+
+        Parameters
+        ----------
+        point (typing.Iterable[float])
```

Comment:
Type annotations will automatically fill out the parameter types in the docs
```suggestion
        point
```

Also as a sidenote, not directly related to this PR, we should probably have a `Point` type hint or something just to unify everything and reduce verbosity

Reply:
Agreed.